src/components/Cards/PreviewCard/PreviewCard.module.css

```diff
@@ -53,18 +53,9 @@
 	display: flex;
 	align-items: center;
 	justify-content: center;
-
 	width: 398.33px;
-	height: 220px;
-
-	background-color: var(--background-purple-card);
-	background: rgba(255, 255, 255, 0.05);
-	background-size: cover;
-	background-position: center;
-	background-repeat: no-repeat;
-
+	max-height: 220px;
 	border-radius: 9px 0px 0px 13px;
```

Comment:
This border radius isn't applying to the image for whatever reason - the image in preview card [should have rounded edges like this](https://www.figma.com/file/0zFJaRNODss8lxwRygUh24/UI-Wilder-Market?node-id=5925%3A144039).